elastic/kibana
# Fixes relative timezone bug #18133 for Chromium reports

This fixes this bug: https://github.com/elastic/kibana/issues/18133 for Chromium reports (but not for Phanom, as [Phantom doesn't give us a good way to set its launch environment variables](https://github.com/teampoltergeist/poltergeist/issues/885)).

TLDR; We launch Chromium with the `TZ` environment variable set to whatever `browserTimezone` is. This should mean that the associated reports will run in the correct timezone.

## Testing

- Create a bunch of docs w/ dates across the past few days:

      POST timebar/_doc
      {
        "foo": "baz", "timestamp": "2018-10-01T10:44:57.305"
      }

- Change the OS time zone (e.g. to somewhere 12 hours away or so)
- Launch Firefox (it usually retains the time zone it was launched with).
- Change the OS time zone back to whatever it was
- Launch Chrome
- Create a dashboard w/ a count of the docs (e.g. a gauge or something)
- Set the time period to "Today"
- Print a report from FF and Chrome... the report should display different values
